PWGDQ/Core/MCSignalLibrary.h

```diff
@@ -219,11 +243,16 @@ MCSignal* o2::aod::dqmcsignals::GetMCSignal(const char* name)
     signal = new MCSignal(name, "muons from jpsi decays", {prong}, {-1});
     return signal;
   }
-  if (!nameStr.compare("eFromPsi2S")) {
+  if (!nameStr.compare("eFromPsi2s")) {
```

Comment:
S is the identifier of the wave state, which normally uses capital S, why change it to small s ?